db/migrate/20170307170911_create_thredded.thredded.rb

```diff
@@ -0,0 +1,244 @@
+# This migration comes from thredded (originally 20160329231848)
```

Comment:
Good, needed to ensure that newer systems don't migrate latest two snapshots when Thredded not yet initialized. An error on my part to have not included this before. Please ensure this is pushed to production.